src/components/MovieTile/MovieTile.tsx

```diff
@@ -14,15 +21,34 @@ const MovieTile: React.FC<MovieTileProps> = ({ movie }) => {
   };
 
   const handleMenuItemSelect = (action: string) => {
-    console.log(action);
     setShowMenu(false);
+
+    switch (action) {
+      case MovieActions.Edit:
+        setShowAddMovieModal(true);
+        break;
+      case MovieActions.Delete:
+        setShowDeleteMovieModal(true);
+        break;
+      default:
+        break;
+    }
   };
 
   const handleTileClick = () => {
     const contextValue = selectedMovie?.id === movie.id ? null : movie
     setSelectedMovie(contextValue);
   };
 
+  const onMovieFormSubmitted = (movie: MovieFormData) => {
+    console.log(movie);
```

Comment:
**nitpick:** I think `console.log` is redundant here

Reply:
done